cmd/create/cluster/cmd.go

```diff
@@ -1758,7 +1758,16 @@ func run(cmd *cobra.Command, _ []string) {
 		reporter.Infof("To view a list of clusters and their status, run 'rosa list clusters'")
 	}
 
-	_, err = ocmClient.CreateCluster(clusterConfig)
+	// Create the access key for the AWS user:
+	awsAccessKey, err := awsClient.GetAWSAccessKeys()
```

Comment:
I think this is going to cause #637 to be unavoidable. Previously STS mode avoided a code path that upserted the credentials. `ocm` usage with `osdCCSadmin` then breaks because it depends on those credentials and they need rotating again to actually get them for use.